src/polar/Pie.tsx

```diff
@@ -647,7 +647,14 @@ export class Pie extends PureComponent<Props, State> {
       !isNumber(innerRadius as number) ||
       !isNumber(outerRadius as number)
     ) {
-      return null;
+      /*
+       * This used to render `null`, but it should still set the legend because:
+       * 1. Hidden pie still renders a legend item (albeit with inactive color)
+       * 2. if a dataKey does not match anything from props.data, then props.sectors are not defined.
+       * Legend still renders though! Behaviour (2) is arguably a bug - and we should be fixing it perhaps?
```

Comment:
confused on the behavior. You say if it doesnt match anything from data then there are no sectors - that makes sense right? No data, no sectors. Where is the bug?

Reply:
Oh.. it still renders even though that is the case. Got it

Reply:
Legend shows but sectors don't.

Reply:
I was also confused 😬